backend/src/controllers/person.controller.ts

```diff
@@ -15,41 +15,67 @@ export const createPerson: POST = async (
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
-  logger.info('POST /persons/create request from frontend');
+  logger.info('POST /persons request from frontend');
+  const auth_id = req.headers.authorization?.["user_id"];
 
   try {
-    const person = await personService.createPerson(req.body);
-    res.status(httpStatus.CREATED).send(person);
-  } catch (e) {
+    let user = await userService.getUserByAuthId(auth_id);
+
+    if (!user) {
+      res.status(httpStatus.NOT_FOUND).end();
+    } else {
+      // Create a new person with the provided information
+      const createdPerson = await personService.createPerson(req.body);
+
+      if (!createdPerson) {
+        res.status(httpStatus.BAD_REQUEST).end();
+      } else {
+        // Add a reference to the created person to the user
+        user = await userService.addPersonId(user.auth_id, createdPerson._id)
+        // If user doesn't contain reference to new person return Conflict
+        if (!user?.persons.includes(new mongoose.Types.ObjectId(createdPerson._id))) {
```

Comment:
Do we want to delete the person from the database if a CONFLICT does happen because of failing to add a reference in the user?

Reply:
Oh true that would be good, will add this in

Reply:
Actually I think #137 is adding a `deletePersons()` function already. I might have to wait for that one to be merged first.